python/src/equistore/operations/empty_like.py

```diff
@@ -21,14 +21,12 @@ def empty_like(
     :param requires_grad: If autograd should record operations for the returned
                           tensor. This option is only relevant for torch.
 
-    Here is an example using this function. First we create a ``TensorMap`` with
-    just one block with two gradients, named ``alpha`` and ``beta``, containing
-    random data for values and gradients.
-
     >>> import numpy as np
     >>> import equistore
     >>> from equistore import TensorBlock, TensorMap, Labels
     >>> np.random.seed(1)
+    >>> # First we create a TensorMap with just one block with two gradients,
+    >>> # named 'alpha' and 'beta', containing random data
```

Comment:
Can we move this above the example? I think this disturbs a bit reading the code.

Reply:
I pushed this as a fixup commit f6b511bb3b9a0f0cc7d56433f4e41d8198f8713c, I'll rebase to merge it with the corresponding commit.